tests/e2e/strapi/.env.example

```
@@ -0,0 +1,12 @@
+# Server
+HOST=0.0.0.0
+PORT=1337
+APP_KEYS=e2e-test-key-1,e2e-test-key-2,e2e-test-key-3,e2e-test-key-4
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APP_KEYS key should go before the HOST key

(UnorderedKey)